<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IstorijaGresaka</class>
 <widget class="QDialog" name="IstorijaGresaka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>531</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrdi</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Smetnje zatezanja trake</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preteze uze desno</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preteze uze lijevo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bocno skretanje rep</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bocno skretanje pog.st.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>preopt.Bimetal M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preopt.Bimetal M2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preg. Os M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grij. rez red M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grij.Nam M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grj.lez M2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grij.Nam M2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SF6 gas M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SF6 gas M2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zapunj.Kosa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not pogon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not kljuc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Proklizavanja</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Krajnji vrata M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Krajnji vrata M2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Greska kocnice sig</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi BAZU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Osvjezi BAZU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>630</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrdi vrijeme</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>451</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>610</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>670</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IstorijaGresaka</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IstorijaGresaka</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
